dfvfs/vfs/attribute.py

```diff
@@ -25,6 +25,8 @@ class StatAttribute(object):
   """Attribute that represents a POSIX stat.
 
   Attributes:
+    device_identifier (Tuple[int, int]): device identifier (if special file),
+        equivalent to st_rdev.
```

Comment:
Let's see if can reword this slightly maybe `comparable to st_rdev`, since st_rdev is of type `dev_t` (which is a 32-bit integer) and not a tuple.

```
stat -c '%R' /dev/console 
501
```

Reply:
an other option is to have the libraries return device identifier instead of device major and minor number

Reply:
As discussed, changed attribute name to device_number.